packages/filecoin-message/src/index.ts

```diff
@@ -24,6 +24,7 @@ export interface LotusMessage {
   GasPremium: string
   GasLimit: number
   GasFeeCap: string
+  GasPrice?: string
```

Comment:
I had to add this to get the tests to pass. Okay?

Reply:
Yeah for now it's fine. I'll look into it later https://github.com/glifio/modules/issues/23